### PR TITLE
Add .htaccess for std.container redirect

### PR DESCRIPTION
**Pull request by CyberShadow**
See comments for details.

### CODE REVIEW

File: posix.mak [hunk 1]
```diff
@@ -124,7 +124,7 @@ STYLES=$(addsuffix .css, $(addprefix css/, \
 PRETTIFY=prettify/prettify.css prettify/prettify.js
 
 PREMADE=appendices.html articles.html fetch-issue-cnt.php	\
-howtos.html language-reference.html robots.txt process.php
+howtos.html language-reference.html robots.txt .htaccess process.php
```
Comment (MartinNowak) [inline posix.mak]: win32.mak?

File: .htaccess [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+RewriteEngine On
+
+# Indicate that this directory maps to the root
+# directory of the domain.
+RewriteBase /
+
+# Phobos rewrites:
+
+# Package files are now generated to std_foo.html
+# instead of std_foo_package.html. This change was
```
Comment (MartinNowak) [inline .htaccess]: Now means in the future? What do I need to pull?

Comment (MartinNowak) [inline .htaccess]: Oh, 1 hour in the past.

File: posix.mak (hunk 1 above)
Comment (CyberShadow) [inline posix.mak]: win32.mak can't build many parts of the website anyway, not nearly enough as needed to the point where you can start it in Apache and get a dlang.org instance.

**Pull request merged**